### ISSUE TITLE
Audio API not available

### ISSUE THREAD

**Issue by jakub-gawlas** [facebook/react-vr#80]
### Description

I hope to add sound effects to my app, unfortunately *VrSoundEffects* has been removed from library. I was tried to use *Web Audio API*, but seems like is not available from scope of React components.

#### Expected behavior

I would like to load sound effect on `componentWillMount` in my component:
```js
  componentWillMount(){
    this.sound = new Audio(asset('effect.mp3').uri);
  }
```

#### Actual behavior
Inside component class, `Audio` is not available in global *window* object

#### Logs
```
Audio is not defined
```

### Solution
Add  `Audio` object to *DedicatedWorkerGlobalScope*?

### Additional Information

* react-vr package version: 0.1.7
* react-vr-web package version: 0.1.7
* Operating System: MacOS
* Browser: Chrome 56.0.2924.87 (64-bit)

**Comment by andrewimm** [comment]
> unfortunately VrSoundEffects has been removed from library

Sorry, but this shouldn't be the case. Can you verify this in some way?

**Comment by andrewimm** [comment]
Also, raw interfacing to the web audio API from web workers, was specc'd out in late 2014, but no browser has implemented it yet. Your noted solution is way beyond the scope of this project, adding `Audio` to `DedicateWorkerGlobalScope` is something only browser vendors can do.

**Comment by jakub-gawlas** [comment]
@andrewimm thanks for explanation :)

> Sorry, but this shouldn't be the case. Can you verify this in some way?

I tried import `VrSoundEffects` in this way:
```js
import { VrSoundEffects } from 'react-vr';
```
Value of `VrSoundEffects` in this case is `undefined`.

**Comment by andrewimm** [comment]
Used to be that you needed to import it via `require('VrSoundEffects')`
Fixed this in 0.2.0, it's now a top-level export of `react-vr`. Thanks for bringing this to my attention.
